src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.hpp

```diff
@@ -83,7 +91,13 @@ class LLMInferRequest : public ov::ISyncInferRequest {
                         ov::SoPtr<ov::ITensor> position_ids,
                         ov::SoPtr<ov::ITensor> input_token_ids);
 
+    // Multiple KV cache model variants with different sizes
+    std::vector<std::shared_ptr<ov::IAsyncInferRequest>> m_generate_requests;
+    std::vector<uint32_t> m_kvcache_sizes;
```

Comment:
Do we need this? We have the same information in the `m_npuw_llm_compiled_model`, right?